### DISCUSSION TITLE
GitHub API create pull request is breaking

### DISCUSSION THREAD

**Discussion by karan8112** [community/community#84676] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

GitHub createPullRequest API has some weird issue, when creating pull request, the forked repo is wrong when invoking the API.
We started observing this issue from Mid December. 

For example:
Url: POST https://hostname/api/v3/repos/{owner}/{repo}/pulls
Header: Authorization:Bearer token
Body:
{
"base": "master",
"body": "test",
"head": "upgrades:update-1",
"head_repo": "{repo}",
"maintainer_can_modify": true,
"title": "pull request"
}
 
But the above post api call create pull request against different repo which is not mentioned in the above url.
Please let me know if there is anything missing.

**Comment by codesmith-emmy** [comment]
May be you should visit GitHub’s [Support Page](https://support.github.com/contact) and open a ticket.
